Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteProjectController.java

```diff
@@ -66,9 +97,12 @@
     @CheckPublicAccess
     @InjectProjectByName
     @RequestMapping(value = "sites/{ProjectUnixName}", method = RequestMethod.GET)
-    public <T> String showProject(Model model, @ProjectIdentifier @PathVariable("ProjectUnixName") String unixName,
+    public String showProject(Model model, @ProjectIdentifier @PathVariable("ProjectUnixName") String unixName,
             Principal principal, @CheckAccess @InjectProject IProject project) throws QuadrigaStorageException {
 
+        String projectId = project.getProjectId();
+        Integer count = 1;
```

Comment:
I wouldn't use a variable for this. You only use it once, right? Also, the variable should be declared where you use it.